light_node.cpp

```diff
@@ -558,9 +558,10 @@ void LightNode::setHaEndpoint(const deCONZ::SimpleDescriptor &endpoint)
                 }
                 else if (i->id() == IDENTIFY_CLUSTER_ID)
                 {
-                    if (manufacturerCode() == VENDOR_IKEA && deviceId == DEV_ID_RANGE_EXTENDER)
+                    if ((manufacturerCode() == VENDOR_IKEA && deviceId == DEV_ID_RANGE_EXTENDER) ||
+                        (manufacturerCode() == VENDOR_EMBER && deviceId == DEV_ID_RANGE_EXTENDER))
```

Comment:
Her you can be specific same as above :)

```cpp
if ((manufacturerCode() == VENDOR_IKEA && deviceId == DEV_ID_RANGE_EXTENDER) ||
    R_GetProductId(this) == QLatin1String("Zigbee Repeater"))

```

Reply:
I need to check too deviceId == DEV_ID_RANGE_EXTENDER for tuya, so better to use 
if (deviceId == DEV_ID_RANGE_EXTENDER && (manufacturerCode() == VENDOR_IKEA || manufacturerCode() == VENDOR_EMBER))